Frameworks/include/CollectionHelpers.h

```diff
@@ -81,7 +81,7 @@ HRESULT WRLToNSCollection<ABI::Windows::Foundation::Collections::IMap<HSTRING, H
         });
     RETURN_IF_FAILED(hr);
 
-    *pDictionary = collection;
+    *pDictionary = [collection autorelease];
```

Comment:
In the other places too


---
In reply to: [107799430](https://github.com/Microsoft/WinObjC/pull/2312#discussion_r107799430) [](ancestors = 107799430)